### ISSUE TITLE
Error when wrong SMTP details are added

### ISSUE THREAD

**Issue by sahiliamsso** [GluuFederation/oxTrust#976]
Configuration > Organization configuration > SMTP Server Configuration
Provide all valid entries except port number.  You wil get below error instead of Error message on screen.

![image](https://user-images.githubusercontent.com/21104811/39385876-b01e5086-4a8f-11e8-8b6b-beaec93e207d.png)

**Comment by syntrydy** [comment]
@sahiliamsso  i don't think this error can be handle from our code. That exception is throw by an upstream server hence difficult to have control over it. A try catch will not work in this case.

**Comment by yurem** [comment]
Hi, Are you testing in Ubuntu 14.04? Can you check if Apache module eavisve is disabled. Davit told that he disable it in final build.
